src/common/errorAlert.js

```diff
@@ -0,0 +1,10 @@
+import { Alert } from 'react-native';
+
+export const showErrorAlert = (message, title) => {
+  Alert.alert(
+           title,
+           message,
+           [{ text: 'OK', onPress: () => {} }],
+           { 'cancelable': true }
```

Comment:
Why ampersands around `cancellable`?

I don't think cancellable: true is correct, as an error message should not have a cancel option.